src/mixxx.cpp

```diff
@@ -598,6 +596,11 @@ void MixxxMainWindow::initializeWindow() {
     Pal.setColor(QPalette::Background, MenuBarBackground);
     m_pMenuBar->setPalette(Pal);
 
+    // restore window size/position
+    QSettings settings("mixxxdj", "mixxx");
```

Comment:
If you really want me to use cfg file then can you please write the code yourself? I'm not particularly motivated to spend my time on it, as the QSettings approach works well, and is cleaner and more appropriate IMHO.